Secret-agency-persistence/src/main/java/cz/fi/muni/pa165/secretagency/entity/Department.java

```diff
@@ -114,6 +120,36 @@ public void setLongitude(Double longitude) {
         this.longitude = longitude;
     }
 
+    /**
+     * @return Department's specialization
+     */
+    public DepartmentSpecialization getSpecialization() {
+        return specialization;
+    }
+
+    /**
+     * Sets specialization of department.
+     * @param specialization Department's specialization
+     */
+    public void setSpecialization(DepartmentSpecialization specialization) {
+        this.specialization = specialization;
+    }
+
+    /**
+     * @return All agents assotiated with this department.
+     */
+    public List<Agent> getAgents() {
+        return Collections.unmodifiableList(this.agents);
+    }
+
+    /**
+     * Adds agent to this department.
+     * @param agent Agent to be added to this department.
+     */
+    public void addAgent(Agent agent) {
+        this.agents.add(agent);
```

Comment:
Pokud budeme dělat obousměrnou vazbu, tak by tu pak ještě mělo být:
agent.setDepartment(this);

Reply:
Ano už si spomínam, hneď to pridám.

Reply:
Teda pridal by som ale Agent nemá vazby pridané.

Reply:
Ještě je možnost, že vazby dáme pouze u agenta. To je asi na nás. Mít je obousměrný je pak pohodlnější pro programátora, ale může dojít ke snížení výkonu programu. U takhle malýho projektu se to ale nejspíš neprojeví. :) 
Takže bych hlasoval spíše pro ty obousměrný.

Reply:
https://github.com/adamkral12/PA165/pull/26

ready for review

Reply:
je to na masteru, muzes upravit @milossilhar